apps/omg_child_chain/lib/omg_child_chain/block_queue.ex

```diff
@@ -182,15 +182,24 @@ defmodule OMG.ChildChain.BlockQueue do
   end
 
   @doc """
-  Lines up a new block for submission. Presumably `OMG.State.form_block` wrote to the `:internal_event_bus` having
-  formed a new child chain block.
+  Lines up a new block for submission.
   """
-  def handle_info({:internal_event_bus, :enqueue_block, %Block{} = block}, state) do
```

Comment:
Can you also remove the other `:enqueue_block` code that's no longer used? https://github.com/omgnetwork/elixir-omg/search?q=%3Aenqueue_block&unscoped_q=%3Aenqueue_block